app/Livewire/SharedVariables/Team/Index.php

```
@@ -40,6 +46,119 @@ public function saveKey($data)
+                }
+            } else {
+                $this->team->environment_variables()->create([
+                    'key' => $key,
+                    'value' => $value,
+                    'is_multiline' => false,
+                    'is_literal' => false,
+                    'type' => 'team',
+                    'team_id' => currentTeam()->id,
+                ]);
+                $count++;
+            }
```

PHPMD (2.15.0)

The method updateOrCreateVariables uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)